docx_editor/track_changes.py

```
@@ -786,11 +786,14 @@ def group_revisions(self, group_id: int) -> tuple[int, ...]:
             )
         return members
 
-    def _resolve_paragraph(self, ref: ParagraphRef):
+    def _resolve_paragraph(self, ref: ParagraphRef, paragraphs: list[Element] | None = None):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_resolve_paragraph`

(ANN202)